File: hunter10.py
a,b=raw_input().split()
a=int(a)
b=int(b)
c=raw_input().split()
d=raw_input().split()
count=0
for i in range(0,b):
	if len(c)==a:
		if d[i] in c:
			count=count+1
if count==b:
	print "YES"
else:
	print "NO"
		

File: hunter10.c
#include<stdio.h>
int main()
{
int n,m,a[25],b[25],i,j,count=0;
printf("\nEnter the how much numbers in A:");
scanf("\n%d",&n);
printf("\nEnter the numbers:");
for(i=0;i<n;i++)
{
   scanf("%d",&a[i]); 
}
printf("\nEnter the how much numbersin B:");
scanf("\n%d",&m);
printf("\nEnter the numbers:");
for(i=0;i<m;i++)
{
   scanf("%d",&b[i]); 
}

for(i=0;i<n;i++)
{
    for(j=i+1;j<n;j++)
    {
    if(b[i]==a[j])
    {
     count=count+1;   
    }
    }
}
if(count<n)
{
    printf("Yes");
}
else
{
    printf("No");
}
return 0;
}
